src/js/view/utils/views-manager.js

```diff
@@ -45,6 +48,24 @@ function scheduleResponsiveRedraw() {
     });
 }
 
+function onOrientationChange() {
+    views.forEach(view => {
+        if (view.model.get('visibility') >= 0.75) {
+            const state = view.model.get('state');
+            const orientation = window.screen.orientation.type;
+            const isLandscape = orientation === 'landscape-primary' || orientation === 'landscape-secondary';
+
+            if (!isLandscape && state === 'paused') {
+                // Set fullscreen to false when going back to portrait while paused and return early
+                view.api.setFullscreen(false);
+                return;
+            } else if (state === 'playing') {
+                view.api.setFullscreen(isLandscape);
```

Comment:
Since only one player can be fullscreen at a time, you should return early here. It would not be good to call `setFullscreen(true)` on two api instances at once.